### PR TITLE
Make landing page default route

### PR DESCRIPTION
**Pull request by aniruddhamurali**
Changed default route from donate page to landing page.

### CODE REVIEW

**Review by PC98 — COMMENTED** [review]
Instead of pushing to `/home`, let's remove the `/home` route and have `pages/index.ts` render the `Home` component.

**Comment by aniruddhamurali** [comment]
Does this mean I just have to `return Home` in `const IndexPage`? I'm a bit confused about how this works.

**Comment by PC98** [comment]
Yep exactly.
You can delete the `pages/home.tsx` file and copy over relevant pieces of code (like the `return <Home` part)  from that file to `pages/index.ts` (please also rename `index.ts` to `index.tsx).

**Comment by aniruddhamurali** [comment]
I'm still having some trouble with this - I think the main difficulty for me is passing in the props for the home page in `index.tsx`.

**Comment by PC98** [comment]
I think you can literally just rename `pages/home.tsx` to `pages/index.tsx` and delete the existing `pages/index.ts` file

**Comment by PC98** [comment]
Once you do that^ feel free to also delete the `getDefaultNonprofitId()` function from our codebase since it won't be used anymore

File: config.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ export default {
   baseUrl: process.env.BASE_URL,
   pages: {
     index: "/",
+    home: "/home",
```
Comment (PC98) [inline config.ts]: nit: delete

**Comment by aniruddhamurali** [comment]
It's hilarious that this was as simple as renaming the file LOL. Thanks so much for the help!

**Review by PC98 — APPROVED** [review]

**Pull request merged**